justfile

```diff
@@ -169,6 +169,11 @@ undeploy:
         echo "Namespace $ns does not exist, nothing to undeploy."
         exit 0
     fi
+    if [[ {{ platform }} == "AKS-PEER-SNP" ]]; then
+        kubectl delete --cascade=foreground -f .//{{ workspace_dir }}/deployment
+        kubectl delete namespace $ns
+        exit 0
```

Comment:
Is there an advantage in delete the namespace here and exiting rather than just falling through to the existing code?